schemas/iso19110/src/main/plugin/iso19110/layout/layout.xsl

```diff
@@ -120,6 +120,70 @@
 
   </xsl:template>
 
+  <!-- Render simple element wi9th gmx:Anchor which usually match a form field -->
```

Comment:
Changes in iso19110 schema are related to this feature? I think no, but probably were committed in the same repository for other feature, please check and if not relevant would be good to remove and send them in other pull request.